app/components/c-product-detail.vue

```diff
@@ -26,7 +26,10 @@
     <section :class="b('bottom')">
 
       <div :class="b('main', { area: 'bottom' })">
-        <div :class="b('details')"> details</div>
+        <div v-if="product.description" :class="b('details')">
+          <e-heading underline tag-name="h2" color="gray">{{ $t('c-product-detail.product-description') }}</e-heading>
```

Comment:
- [ ] Would name the translation `c-product-detail.product-description-title`
How many different heading colors we have? If there are more than 2 I would recommend to use here color codes